src/pkg/cli/getDeploymentStatus.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/DefangLabs/defang/src/pkg/cli/client"
+	"github.com/DefangLabs/defang/src/pkg/term"
+)
+
+func WaitCdTaskState(
+	ctx context.Context,
+	provider client.Provider,
+) error {
+	term.Debug("waiting for cdTask to complete.\n") // TODO: don't print in Go-routine
+
+	if DoDryRun {
+		return ErrDryRun
+	}
```

Comment:
Should we push this up into the caller? If we know it's a dry run, we probably don't even need to invoke `WaitCdTaskState`, (and print the debug message above) right?